fleet/view/js/create_fleet.js

```diff
@@ -0,0 +1,384 @@
+var loc = window.location.pathname;
+
+var split_url = loc.split('/');
+var dir = split_url[2];
+var url_host = window.location.origin + "/" + split_url[1] + "/" + split_url[2];
+var loc_host = window.location.origin
+var split_host = loc_host.split('/');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/davidgunadi/hello-world/pullRequest?prid=8400071)
```suggestion
var split_host = loc_host.split("/");
```